public/catch-js-errors.js

```diff
@@ -118,6 +121,23 @@
 					savedWindowOnError();
 				}
 			};
+		} else if ( localStorageAssignment === 'analytics' ) {
+			window.onerror = function( message, scriptUrl, lineNumber ) {
+				if ( typeof window.ga === 'function' ) {
+					window.ga(
```

Comment:
Related documentation for ga events https://developers.google.com/analytics/devguides/collection/analyticsjs/events#overview